src/AddIns/Uno.UI.Lottie/LottieVisualSourceBase.Android.cs

```diff
@@ -260,3 +261,62 @@ private Size CompositionSize
 		}
 	}
 }
+#else
+using System;
+using System.Threading;
+using Windows.Foundation;
+using Windows.UI.Xaml.Controls;
+using System.Threading.Tasks;
+using Uno.Disposables;
+
+namespace Microsoft.Toolkit.Uwp.UI.Lottie
+{
+	partial class LottieVisualSourceBase
+	{
+		public bool UseHardwareAcceleration { get; set; } = true;
+
+		async Task InnerUpdate(CancellationToken ct)
+		{
+
+		}
+
+		public void Play(double fromProgress, double toProgress, bool looped)
+		{
+			throw new NotImplementedException();
```

Comment:
There's an issue for that?

Reply:
Lottie may work soon, it depends on how upcoming previews will handle monoandroidXX.XX targets. I'll link to the .NET 6 support epic.